### ISSUE TITLE
Is it better to distinguish reserved fields with distinct syntax?

### ISSUE THREAD

**Issue by da4089** [finspec/finspec-spec#5]
There are often reserved fields in message definitions, usually with a "must be zero" or "must be spaces" stipulation in the spec text.

Of course, these can be represented within the fields list, but eg. the _name_, _description_, _type_, and _format_ properties are more-or-less redundant, and rather than all the verbiage of a single-valued enum set, it might be better to just have a _fill_ attribute?

As an example, I'm thinking of replacing a field specification like this:

```
                {
                    "position": 30,
                    "name": "Reserved Field2",
                    "description": "This will always be 0",
                    "type": "integer",
                    "format": "int8",
                    "length": 1,
                    "values": [
                        {
                            "enum": 0,
                            "description": "0"
                        }
                    ]
                }
```
with something like

```
                {
                    "position": 30,
                    "type": "reserved",
                    "length": 1,
                    "fill": 0
                }
```

This makes the semantics much more clear, which should help when generating encoder/decoder routines, etc.

**Comment by CJLees01** [comment]
:+1:

**Comment by mahenpatil** [comment]
Included in v0.2. (Define reserved datatype and point datatype of the field to the same).
